### PR TITLE
DOC: add generator example

### PR DESCRIPTION
**Pull request by ValeryVal**
Original pull request fix: https://github.com/numpy/numpy/pull/22560 

Co-authored-by: aviralsomani <avisomani@gmail.com>

### CODE REVIEW

**Comment by rkern** [comment]
Please follow the [docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html#examples) for formatting the examples properly. There needs to be blank spaces between the interpreter input/output and the prose comments.

Please don't use the Greek characters. I'm not sure that the backslashes will do what you want outside of the `.. math` block, in any case, and I'd recommend referring to the actual parameter names `a` and `b`.

Please do not use `np.random.beta()` in the example. That function is irrelevant to documenting `Generator`. See the other examples in `Generator` methods. E.g.

```
>>> rng = np.random.default_rng()
>>> rng.beta(0.5, 0.5, 3)
...
```

**Comment by codeguru42** [comment]
@rkern I was helping @ValeryVal with this PR at PyCon 2023. I see that our most recent changes failed some of the CI checks. But I'm having difficulty deciphering the errors. Are you able to provide any insight into what we need to change to get this over the line?

**Comment by mattip** [comment]
The f2py problem with 32-bit windows is not connected to this PR.

**Comment by codeguru42** [comment]
@rkern @mattip Is there anything else that needs to be done here?

File: numpy/random/_generator.pyx [hunk 1]
```diff
@@ -396,6 +396,26 @@ cdef class Generator:
...
+        and `b` respectively below
+        Vector values for `a` and `b`, results in 3 * 2 ndarray
+        with singular samples for each pairwise combination
+        of `a` and `b`
+
+        >>> rng = np.random.default_rng()
+        >>> rng.beta([[0.5], [0.25], [0.75]], [[0.5, 0.25]])
+        array([[6.26304645e-01, 3.57283854e-02],
+                [1.40768012e-02, 1.72984613e-05],
+                [4.63162592e-01, 3.38723747e-01]])  # random
```
Comment (rkern) [inline numpy/random/_generator.pyx:418]:
```suggestion
```

I would delete this paragraph and the following example. The broadcasting behavior that's being demonstrated is general to all of the methods, and we don't really need to demonstrate it just for `beta`. And "sublist" is a novel term that we don't use anywhere else; not entirely sure what that's referring to.

What's left doesn't really add much. Generally, in these examples, we try to use the example to point out something interesting about the relationship between the parameters and the generated values (e.g. that we can reproduce the mean and variance of a normal distribution). A kind of minimum would be to [plot a histogram](https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.triangular.html).

**Comment by melissawm** [comment]
Hi @ValeryVal - gentle ping, are you still interested in finishing this up? Cheers!

**Comment by ValeryVal** [comment]
> Hi @ValeryVal - gentle ping, are you still interested in finishing this up? Cheers!

Hi, @melissawm! I'd like to pass this issue to someone more specialized in the subject. Thank you!

**Review by rossbar — COMMENTED** [review]
I will go ahead and close this one given the feedback and the bandwidth to continue. Thanks all!

**Pull request closed without merging**